types/cloudfour__simple-svg-placeholder/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for @cloudfour/simple-svg-placeholder 1.0
+// Project: https://github.com/cloudfour/simple-svg-placeholder#readme
+// Definitions by: Mike Deverell <https://github.com/devrelm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function simpleSvgPlaceholder(options?: {
+    width?: number;
+    height?: number;
+    text?: string;
+    fontFamily?: string;
+    fontWeight?: string | number;
+    fontSize?: number;
+    dy?: number;
+    bgColor?: string;
+    textColor?: string;
+    dataUri?: boolean;
+    charset?: string;
```

Comment:
```suggestion
    width?: number | undefined;
    height?: number | undefined;
    text?: string | undefined;
    fontFamily?: string | undefined;
    fontWeight?: string | number | undefined;
    fontSize?: number | undefined;
    dy?: number | undefined;
    bgColor?: string | undefined;
    textColor?: string | undefined;
    dataUri?: boolean | undefined;
    charset?: string | undefined;
```
It looks library uses destructuring parameter assignments with default values:
https://github.com/cloudfour/simple-svg-placeholder/blob/3bbbb1de559b1ddc41501a142e01b998348c0fe0/index.js#L1-L13
So types can support exact optional property types (which we advice to add for better interop):
https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types

```ts
// if written in native TS
function simpleSvgPlaceholder({
    width = 300,
    height = 150
}: { width?: number | undefined; height?: number | undefined } = {}) {

    const str = `<svg xmlns="http://www.w3.org/2000/svg" width="${width}" height="${height}" viewBox="0 0 ${width} ${height}">
  </svg>`;
    return str;
}

simpleSvgPlaceholder({
    width: undefined,
    height: undefined,
});
```